packages/backend/src/controllers/house/dto/create-house.dto.ts

```diff
@@ -10,4 +11,7 @@ export class CreateHouseDto extends HouseModel {
 
   @ApiProperty()
   readonly address: string;
+
+  @ApiProperty()
+  users: Array<Types.ObjectId>;
```

Comment:
This is not needed in the create house dto if we are only adding the requesting user as a user of the house in the create house endpoint.